spec/data-model/README.md

```diff
@@ -266,6 +266,7 @@ interface MarkupClose {
   type: "markup";
   kind: "close";
   name: string;
+  options?: Option[];
```

Comment:
One of the presented reasons for including options on close is that this removes the differences between closing markup and other parts. So we can simplify all of the constructs here into:
```ts
interface Markup {
  type: "markup";
  kind: "open" | "standalone" | "close";
  name: string;
  options: Option[];
  attributes: Attribute[];
}
```

Reply:
Sounds like @mihnita would prefer this form as well: https://github.com/unicode-org/message-format-wg/pull/649#issuecomment-1945282227.

Reply:
Done